msi/release.sh

```diff
@@ -1,17 +1,56 @@
 #!/bin/sh
 
-# Build Git for Windows' .msi files
+me=$(basename $0)
+function showUsage() {
+	cat <<-EOF
```

Comment:
You probably want to remove the `-` from `-EOF` because your careful indentation will otherwise be killed on displaying the usage.